tools/data_load.py

```diff
@@ -0,0 +1,67 @@
+import io
+from sys import argv
+import os
+
+from googleapiclient.discovery import build
+from googleapiclient.http import MediaIoBaseDownload
+from google.oauth2 import service_account
+
+
+CREDENTIALS_PATH = 'tools/gdrive_credentials.json'
```

Comment:
```suggestion
CREDENTIALS_PATH = 'gdrive_credentials.json'
```

gdrive_credentials will be in the same folder as this script (at least if I understood the README correctly). So, revious version doesn't work.

Reply:
changed